### PR TITLE
[zh] sync service storage-classes configure-pdb

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/concepts/storage/storage-classes.md [hunk 1]
```diff
@@ -560,9 +560,19 @@ using `allowedTopologies`.
...
+[GCE CSI Migration](/docs/concepts/storage/volumes/#gce-csi-migration) is
+enabled, a GCE PD volume can be provisioned in a topology that does not match
+any nodes, but any pod trying to use that volume will fail to schedule. With
+legacy pre-migration GCE PD, in this case an error will be produced
+instead at provisioning time. GCE CSI Migration is enabled by default beginning
+from the Kubernetes 1.23 release.
 -->
 `zone` 和 `zones` 已被弃用并被 [allowedTopologies](#allowed-topologies) 取代。
+当启用 [GCE CSI 迁移](/zh-cn/docs/concepts/storage/volumes/#gce-csi-migration)时，
+可以在与任何节点都不匹配的拓扑中提供 GCE PD 卷，但任何试图使用该卷的 Pod 都会失败。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/storage/storage-classes.md:573]:
```suggestion
GCE PD 卷可以在与任何节点都不匹配的拓扑中进行制备，但所有试图使用该卷的 Pod 都会调度失败。
```

File: content/zh-cn/docs/concepts/storage/storage-classes.md [hunk 2]
```diff
@@ -560,9 +560,19 @@ using `allowedTopologies`.
...
+对于传统的迁移前 GCE PD，这种情况下将在配置时产生错误。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/storage/storage-classes.md:574]:
```suggestion
对于传统的迁移前 GCE PD，在这种场景下将在制备时报错。
```

File: content/zh-cn/docs/concepts/storage/storage-classes.md [hunk 3]
```diff
@@ -560,9 +560,19 @@ using `allowedTopologies`.
...
+从 Kubernetes 1.23 版本开始，默认启用 GCE CSI 迁移。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/storage/storage-classes.md:575]:
```suggestion
从 Kubernetes 1.23 版本开始，GCE CSI 迁移默认启用。
```

File: content/zh-cn/docs/tasks/run-application/configure-pdb.md [hunk 4]
```diff
@@ -509,22 +509,26 @@ Pods in `Pending`, `Succeeded` or `Failed` phase are always considered for evict
...
+or with {{< glossary_tooltip term_id="CustomResourceDefinition" text="custom resources" >}}
+that implement a `scale` [subresource](/docs/concepts/extend-kubernetes/api-extension/custom-resources/#advanced-features-and-flexibility),
+and where the PDB selector exactly matches the selector of the Pod's owning resource.
 -->
-## 任意控制器和选择算符   {#arbitrary-controllers-and-selectors}
+## 任意工作负载和任意选择算符   {#arbitrary-controllers-and-selectors}
 
-如果你只使用与内置的应用控制器（Deployment、ReplicationController、ReplicaSet 和 StatefulSet）
-对应的 PDB，也就是 PDB 的选择算符与 控制器的选择算符相匹配，那么可以跳过这一节。
+如果你只在内置的工作负载资源（Deployment，ReplicaSet，StatefulSet 和 ReplicationController）
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/run-application/configure-pdb.md:522]: 中文的枚举用顿号，不用逗号。

File: content/zh-cn/docs/tasks/run-application/configure-pdb.md [hunk 5]
```diff
@@ -509,22 +509,26 @@ Pods in `Pending`, `Succeeded` or `Failed` phase are always considered for evict
...
+或在实现 `scale` [子资源](/zh-cn/docs/concepts/extend-kubernetes/api-extension/custom-resources/#advanced-features-and-flexibility)
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/run-application/configure-pdb.md:523]:
```suggestion
或在实现了 `scale` [子资源](/zh-cn/docs/concepts/extend-kubernetes/api-extension/custom-resources/#advanced-features-and-flexibility)
```

File: content/zh-cn/docs/tasks/run-application/configure-pdb.md (hunk 4 above)
Comment (tengqm) [inline content/zh-cn/docs/tasks/run-application/configure-pdb.md:522]:
```suggestion
如果你只针对内置的工作负载资源（Deployment，ReplicaSet，StatefulSet 和 ReplicationController）
```

File: content/zh-cn/docs/tasks/run-application/configure-pdb.md [hunk 6]
```diff
@@ -509,22 +509,26 @@ Pods in `Pending`, `Succeeded` or `Failed` phase are always considered for evict
...
+的{{< glossary_tooltip term_id="CustomResourceDefinition" text="自定义资源" >}}中使用 PDB，
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/run-application/configure-pdb.md:524]:
```suggestion
的{{< glossary_tooltip term_id="CustomResourceDefinition" text="自定义资源" >}}使用 PDB，
```

File: content/zh-cn/docs/tasks/run-application/configure-pdb.md [hunk 7]
```diff
@@ -509,22 +509,26 @@ Pods in `Pending`, `Succeeded` or `Failed` phase are always considered for evict
...
+并且 PDB 选择器与 Pod 所属资源的选择器完全匹配，那么可以跳过这一节。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/run-application/configure-pdb.md:525]:
```suggestion
并且 PDB 选择算符与 Pod 所属资源的选择算符完全匹配，那么可以跳过这一节。
```

File: content/zh-cn/docs/tasks/run-application/configure-pdb.md [hunk 8]
```diff
@@ -509,22 +509,26 @@ Pods in `Pending`, `Succeeded` or `Failed` phase are always considered for evict
...
 
 <!--
-You can use a PDB with pods controlled by another type of controller, by an
+You can use a PDB with pods controlled by another resource, by an
 "operator", or bare pods, but with these restrictions:
 -->
-你可以使用这样的 PDB：它对应的 Pod 可能由其他类型的控制器控制，可能由 "operator" 控制，
+你可以使用这样的 PDB：它对应的 Pod 可能由其他资源，可能由 "operator" 控制，
 也可能为“裸的（不受控制器控制）” Pod，但该类 PDB 存在以下限制：
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/run-application/configure-pdb.md:532]:
```suggestion
你可以针对由其他资源、某个 "operator" 控制的或者“裸的（不受控制器控制）” Pod
使用 PDB，但存在以下限制：
```

File: content/zh-cn/docs/tasks/run-application/configure-pdb.md [hunk 9]
```diff
@@ -535,11 +539,16 @@ You can use a PDB with pods controlled by another type of controller, by an
...
+It is not possible to use other availability configurations,
+because Kubernetes cannot derive a total number of pods without a supported owning resource.
+
+You can use a selector which selects a subset or superset of the pods belonging to a
+workload resource. The eviction API will disallow eviction of any pod covered by multiple PDBs,
 so most users will want to avoid overlapping selectors. One reasonable use of overlapping
 PDBs is when pods are being transitioned from one PDB to another.
 -->
-你可以令选择算符选择一个内置控制器所控制 Pod 的子集或父集。
+不可能使用其他的可用性配置，因为如果没有支持的所有者资源，Kubernetes 无法推导出 Pod 的总数。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/run-application/configure-pdb.md:550]:
```suggestion
你无法使用其他的可用性配置，因为如果没有被支持的属主资源，Kubernetes 无法推导出 Pod 的总数。
```

File: content/zh-cn/docs/tasks/run-application/configure-pdb.md [hunk 10]
```diff
@@ -535,11 +539,16 @@ You can use a PDB with pods controlled by another type of controller, by an
...
+
+你可以使用一个选择算符，这个选择算符能选择属于工作负载资源的 Pod 的子集或超集。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/run-application/configure-pdb.md:552]:
```suggestion
你可以使用能够选择属于工作负载资源的 Pod 的子集或超集的选择算符。
```

File: content/zh-cn/docs/tasks/run-application/configure-pdb.md [hunk 11]
```diff
@@ -535,11 +539,16 @@ You can use a PDB with pods controlled by another type of controller, by an
...
 驱逐 API 将不允许驱逐被多个 PDB 覆盖的任何 Pod，因此大多数用户都希望避免重叠的选择算符。
-重叠 PDB 的一种合理用途是当 Pod 从一个 PDB 过渡到另一个 PDB 时再使用。
+重叠 PDB 的一种合理用途是当 Pod 从一个 PDB 过渡到另一个 PDB 时再使用。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/run-application/configure-pdb.md:554]:
```suggestion
重叠 PDB 的一种合理用途是将 Pod 从一个 PDB 转交到另一个 PDB 的场合。
```

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 12]
```diff
@@ -195,15 +195,17 @@ spec:
...
+ClusterIP [service type](#publishing-services-service-types). The Service
 targets TCP port 9376 on any Pod with the `app.kubernetes.io/name: MyApp` label.
 
 Kubernetes assigns this Service an IP address (the _cluster IP_),
 that is used by the virtual IP address mechanism. For more details on that mechanism,
 read [Virtual IPs and Service Proxies](/docs/reference/networking/virtual-ips/).
 -->
-应用上述清单将创建一个名称为 "my-service" 的新 Service，它在所有 Pod 上指向
-TCP 端口 9376，并且具有标签 `app.kubernetes.io/name: MyApp`。
+应用上述清单将创建一个名称为 "my-service" 的新服务，
```
[the remark this answers is not in the document]
Comment (asa3311) [reply content/zh-cn/docs/concepts/services-networking/service.md]: Thank you for reviewing my PR, I have learned a lot.

**Pull request merged**